src/_includes/default-layout.njk

```diff
@@ -51,7 +51,7 @@
     <nav id="headerNav" class="header">
         <ul id="navButtons" class="nav-buttons">
             <li><a href="/" class='code home {{ "active" if "/" == page.url }}'>Home</a></li>
-            <li><a href="/schedule/" class='code {{ "active" if "/schedule/" in page.url }}'>Schedule</a></li>
+            {# <li><a href="/schedule/" class='code {{ "active" if "/schedule/" in page.url }}'>Schedule</a></li> #}
```

Comment:
So one thing that is a bummer about nav toggle (hamburger) that starts showing up when the nav items start wrapping is that it's not related to when the nav items start wrapping at all.  It's just a media query and it shows up below 740px wide.  Now with this schedule item removed, the items don't start wrapping until ~662px wide.  So the media query has to be changed any time we change the mix of nav items, or we just live with that button showing up for no reason at certain widths.  I'm sure we could come  up with a better way at some point, but just something to be aware of for now.

Reply:
ohh, that makes sense as to what's driving the toggle visibility - yeah, let's just keep track of for now.  I think the goal is to publish speaker / session info before a finalized schedule is built, so I'll push some style changes